Description: Remove the build date from the Javadoc files to make the build reproducible
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no
--- a/build.xml
+++ b/build.xml
@@ -256,6 +256,7 @@
     <javadoc packagenames="${packages}" sourcepath="${basedir}/${src.dir}"
              destdir="${javadoc.dir}" author="true" version="true"
              windowtitle="${Name} ${version} API" doctitle="${Name}"
+             additionalparam="-notimestamp"
              footer="Copyright &amp;copy; 2000-2008 Russell Gold. See &lt;a target=&quot;_top&quot; href=&quot;../license.html&quot;&gt;license agreement&lt;/A&gt; for rights granted." >
         <classpath refid="base.classpath"/>
     </javadoc>
